nss-day-cohort-41/react-nutshell-thepecans
# Add friends' events to event list

# Description
Added friends' events to the user's event list. Made sure the first event of the current user is highlighted. Styled friends' events per ticket instructions. Edit and delete buttons only available on current user's events.

Fixes # 7 (partial)
## Type of change
Please delete options that are not relevant

# Testing Instructions
Make sure your database has events and friends for the current user to test, as well as events on that user's friends. Navigate to "../events". Make sure all events are properly styled (friends' events should be italic and cornsilk; first event for the current user should be larger and highlighted). Make sure edit and delete buttons are only on the current user's events.

# Checklist:
